### PR TITLE
array_pad on list+explode

### PR DESCRIPTION
**Pull request by bucha**
This fixes some edge-cases that we encountered during unit-tests, where the response might not carry a body for example.

### What has been done

-
-

### How to test

-
-

### Todo

- [ ]
- [ ]

### Notes

-
-

### CODE REVIEW

File: src/VCR/Util/HttpUtil.php [hunk 1]
```diff
@@ -66,8 +66,8 @@ public static function parseStatus($status)
     public static function parseResponse($response)
     {
         $response = str_replace("HTTP/1.1 100 Continue\r\n\r\n", '', $response);
-            
-        list($rawHeader, $rawBody) = explode("\r\n\r\n", $response, 2);
+
+		list($rawHeader, $rawBody) = array_pad(explode("\r\n\r\n", $response, 2), 2, '');
```
Comment (MrRio) [inline src/VCR/Util/HttpUtil.php]: Accidental indentation?

**Comment by MrRio** [comment]
This is missing a unit test and those items from the PR template.

**Comment by bucha** [comment]
Yupp. I’ve done this from the Github editor.
This might have caused it.

**Comment by higidi** [comment]
Closing as stalled — superseded by PR #302.

PR #302 is the canonical fix for `HttpUtil::parseResponse` handling responses without a body separator. The fix is also related to the open issue #272. Please follow up in PR #302 if you'd like to contribute.
